osquery/tables/system/windows/services.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <Winsvc.h>
+#include <string>
+
+#include "osquery/tables/system/windows/registry.h"
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#pragma comment(lib, "Advapi32.lib")
```

Comment:
I see we are doing this in `users.cpp`, we should correct that, I'd prefer us collect all of the linking dependencies in the CMake logic. Take a look at `./osquery/CMakeLists.txt` and use the form:
```cmake
ADD_OSQUERY_LINK_ADDITIONAL("Advapi32.lib")
```